src/nodes.coffee

```diff
@@ -3327,9 +3343,8 @@ exports.Assign = class Assign extends Base
   isStatement: (o) ->
     o?.level is LEVEL_TOP and @context? and (@moduleDeclaration or "?" in @context)
 
-  checkAssignability: (o, varBase) ->
-    if Object::hasOwnProperty.call(o.scope.positions, varBase.value) and
-       o.scope.variables[o.scope.positions[varBase.value]].type is 'import'
```

Comment:
Did we lose this `checkAssignability` method?

Reply:
Nope this diff is a combination of renaming `checkAssignability` -> `checkNameAssignability` (in order to disambiguate naming-wise from the added `checkAssignability` option below) as well as simplifying the scope checking by using `o.scope.type()`